Hearthstone Deck Tracker/LogReader/Handlers/TagChangeActions.cs

```diff
@@ -65,10 +65,27 @@ public Action FindAction(GameTag tag, IGame game, IHsGameState gameState, int id
 					return () => OnCardCopy(id, value, game, gameState);
 				case TAG_SCRIPT_DATA_NUM_1:
 					return () => OnTagScriptDataNum1(id, value, game, gameState);
+				case REBORN:
+					return () => RebornChange(id, value, game);
 			}
 			return null;
 		}
 
+		private void RebornChange(int id, int value, IGame game)
+		{
+			if(value != 1)
+				return;
+			foreach(var x in game.Entities)
```

Comment:
This can be simplified as
```
if(game.Entities.TryGetValue(id, out var entity) && entity != null)
    BobsBuddyInvoker.GetInstance(game.CurrentGameStats.GameId, game.GetTurnNumber())?
				.SetMinionReborn(id);
```

However, why are you checking that the entity exists in the first place? What you are passing to `MinionGainedReborn` is just `id`.